types/google__maps/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for @google/maps 0.5
 // Project: https://github.com/googlemaps/google-maps-services-js
 // Definitions by: Indri Muska <https://github.com/indrimuska>
+//                 Gabe O'Leary <https://github.com/goleary>
```

Comment:
Please remove this, usually only the main maintainer should be added, not all contributors.

Reply:
done